### ISSUE TITLE
Does this library really help in solving OutOfMemory issues?

### ISSUE THREAD

**Issue by xstreamcl** [kfiroo/react-native-cached-image#142]

**Comment by chrusart** [comment]
Why it should be? It's caching library, right?
If you get OutOfMemory using lot of images than it's your app or components problem.

**Comment by xstreamcl** [comment]
@chrusart Fair enough. I am working on a react native android application from last 8 months. Very recently app has started facing a lot of these error. I am profiling the app on android studio 3.2 and I can see memory usage and allocated memory at given time. However I don't understand what's wrong.  Any guidance here is appreciated :)

**Comment by chrusart** [comment]
Well, every code/library can have memory leaks, this one too, but I cannot help you here as it happening on your app and how could I find the reason. As I am using this library and if you could shed light where probably are some leaks, then we could find the solution for it. For now I didn't find any OutOfMemory issues (at least not major) that would catch my attention according to this library.
